tests/lib/log.py

```diff
@@ -0,0 +1,23 @@
+import logging
```

Comment:
I made the change to use `logging` as close to recommendation as possible. `log` module is there to provide helper functions and functions setting defaults. `LogHelper` class is a singleton so whole tool has one place where to get `LOG_DIR` and one place to get log preloading.